### PR TITLE
ckeditor touch fixes

### PR DESCRIPTION
**Pull request by vxsx**
Fixes things for touch devices
- [x] possibility to edit nested plugin
- [x] possibility to scroll panels like style/format/cmsplugins
- [x] possibility to scroll inner ckeditor modal when adding/editing plugin
- [x] possibility to resize the editor
- [x] possibility to resize inner ckeditor modal when adding/editing plugin
- [x] possibility to move inner ckeditor modal when adding/editing plugin
- [x] maximize ckeditor if it's alone in the cms modal on touch devices
- [x] make ckeditor dialog open to fit almost full ckeditor instead of 600x200px
- [x] doesn't break existing stuff for desktop
- [x] adds changelog

---------------------------------------------------------------------------
will update ckeditor to 4.5.4 (in next PR not to clutter things here)

### CODE REVIEW

File: djangocms_text_ckeditor/templates/cms/plugins/text_plugin_change_form.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {% block extrahead %}
 {{ block.super }}
-<script src="{% static "cms/js/modules/cms.base.js" %}"></script>
+<script src="{% static "cms/js/dist/bundle.admin.base.min.js" %}"></script>
```
Comment (vxsx) [inline djangocms_text_ckeditor/templates/cms/plugins/text_plugin_change_form.html]: this is a breaking change as it's going to be only available in 3.2+

Comment (FinalAngel) [inline djangocms_text_ckeditor/templates/cms/plugins/text_plugin_change_form.html]: can we add it to the changelog?

Comment (vxsx) [inline djangocms_text_ckeditor/templates/cms/plugins/text_plugin_change_form.html]: i'd love to, but there is no changelog

Comment (FinalAngel) [inline djangocms_text_ckeditor/templates/cms/plugins/text_plugin_change_form.html]: than let's create one :)

File: djangocms_text_ckeditor/static/djangocms_text_ckeditor/ckeditor_plugins/cmsplugins/plugin.js [hunk 2]
```diff
@@ -58,15 +58,25 @@ $(document).ready(function () {
...
+					var element = selection.getSelectedElement() || selection.getCommonAncestor().getAscendant('a', true);
+				}
+				if(element && element.getAttribute('id').indexOf('plugin_obj_') === 0) {
 					event.data.dialog = '';
 					that.editPlugin(element);
 				}
+			}
+			this.editor.on('doubleclick', handleEdit);
+			this.editor.on('instanceReady', function () {
+				CMS.$('img[id*="plugin_obj_"]', CMS.$('iframe.cke_wysiwyg_frame')[0].contentWindow.document.documentElement).on('touchend', handleEdit);
```
Comment (FinalAngel) [inline djangocms_text_ckeditor/static/djangocms_text_ckeditor/ckeditor_plugins/cmsplugins/plugin.js]: thats a but long

File: djangocms_text_ckeditor/static/djangocms_text_ckeditor/css/cms.ckeditor.css [hunk 3]
```diff
@@ -17,4 +17,63 @@
 .cke_button__cmsplugins .cke_button_label { padding-left:5px; padding-right:5px;  display:inline; }
 
 /* make dropdow larger */
-.cke_combopanel { min-width:200px; }
\ No newline at end of file
+.cke_combopanel, .cke_panel { min-width:200px; }
```
Comment (FinalAngel) [inline djangocms_text_ckeditor/static/djangocms_text_ckeditor/css/cms.ckeditor.css]: also break down

File: djangocms_text_ckeditor/static/djangocms_text_ckeditor/css/cms.ckeditor.css [hunk 4]
```diff
@@ -17,4 +17,63 @@
...
+
+
+/* add / edit plugin modal */
+.cms-ckeditor-dialog .cke_dialog_contents_body {
+    padding: 3px 0 !important;
+}
+.cms-ckeditor-dialog .cke_dialog_page_contents {
+    width: 100% !important;
+    /* height: 100% !important; */
```
Comment (FinalAngel) [inline djangocms_text_ckeditor/static/djangocms_text_ckeditor/css/cms.ckeditor.css]: remove comment

File: djangocms_text_ckeditor/static/djangocms_text_ckeditor/css/cms.ckeditor.css [hunk 5]
```diff
@@ -17,4 +17,63 @@
...
+    height: 100% !important
+}
+.cms-ckeditor-dialog .cke_dialog_page_contents table tbody,
+.cms-ckeditor-dialog .cke_dialog_page_contents table tr,
+.cms-ckeditor-dialog .cke_dialog_page_contents table td {
+    display: block;
+    height: 100%;
+}
+.cms-ckeditor-dialog .cke_dialog_page_contents table .cke_dialog_ui_vbox_child {
+    padding: 0 !important;
```
Comment (FinalAngel) [inline djangocms_text_ckeditor/static/djangocms_text_ckeditor/css/cms.ckeditor.css]: order

**Comment by FinalAngel** [comment]
awesome :+1: I guess the codestyle will be cleaned up later anyways - can merge

**Comment by vxsx** [comment]
found a bug on windows surface :/

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**